app/database/models.py

```diff
@@ -42,6 +42,7 @@ class User(Base):
     avatar = Column(String, default="profile.png")
     telegram_id = Column(String, unique=True)
     is_active = Column(Boolean, default=False)
+    privacy = Column(String, default="Private", nullable=False)
```

Comment:
The privacy is per user or per event?

Reply:
Per user. 
Per event is another feature :)